setup.py

```diff
@@ -19,13 +19,23 @@
 import os
 from distutils.core import setup, Extension
 from distutils.command.build import build
+import re
 import subprocess
 import versioneer
 
+EXCLUDE = [
```

Comment:
Can probably remove all of the rewrite code.

Reply:
I put it back as a function to find the glpk.h header and then use swig_opts in the Extension() to include this dir.

This works ok for me (macOS), but the Windows CI builds fail because the header is found in the source dir, but then swig can't find it. I don't know windows or appveyor well enough to resolve the windows / ci issue.